cl/lib/model_helpers.py

```diff
@@ -536,6 +536,28 @@ def linkify_orig_docket_number(agency: str, og_docket_number: str) -> str:
             )
             return f"https://www.nlrb.gov/case/{formatted_number}"
 
+    # US Tax Court pattern
+    if "Tax" or "Internal Revenue" in agency:
```

Comment:
I think the if statement's condition is always true. We can replace it with a different expression:

```suggestion
    if any(x in agency for x in ("Tax", "Internal Revenue")):
```

Reply:
Thanks @ERosendo, fixed.